src/components/templates/Jazzbar/JazzTab/JazzTab.tsx

```diff
@@ -57,6 +66,9 @@ const Jazz: React.FunctionComponent<PropsType> = ({ setUserList, venue }) => {
 
   const venueToUse = venue ? venue : firestoreVenue;
 
+  const jazzbarConfig = venueToUse?.config?.jazzbarConfig;
+  const jazzbarTables = jazzbarConfig?.tables ?? JAZZBAR_TABLES;
```

Comment:
This should be generic across templates. So we could do `venue.config.tables` for example. There are other places that would be great to incorporate into this PR so we have a single way to specify table config and the config will apply regardless of any template changes.

Reply:
My thoughts with this are that it would be nice to be able to have more tightly constricted types where we know what is expected to be possible, and what isn't. That lead me to this implementation originally.

But thinking about it really quickly now we should be able to do this even if it is more generically named (eg. using type descriminators based on the string value of template or similar, OOS of this PR), so I support @thecadams 's view.

Reply:
resolved in https://github.com/sparkletown/sparkle/pull/793/commits/df6c5ff6e1ce46d62bfb2cb0ec97cec7f0ea0e65